### PR TITLE
feat: Add command to generate sources tables

### PR DESCRIPTION
**Pull request by erezrokah**
#### Summary

Needs https://github.com/cloudquery/plugin-sdk/pull/726.
Fixes https://github.com/cloudquery/cloudquery/issues/8731

### CODE REVIEW

File: cli/go.mod [hunk 1]
```diff
@@ -3,7 +3,7 @@ module github.com/cloudquery/cloudquery/cli
 go 1.19
 
 require (
-	github.com/cloudquery/plugin-sdk v1.42.0
+	github.com/cloudquery/plugin-sdk v1.42.1-0.20230307170513-f5cd8633c7a0
```
Comment (erezrokah) [inline cli/go.mod:6]: This points to a commit sha https://github.com/cloudquery/plugin-sdk/pull/726/commits/f5cd8633c7a0cfb3e8f9c77bdea67df4d411c150.

We should use the official version before merging

**Comment by hermanschaaf** [comment]
What happens if there are two different versions of the same source plugin specified in the spec? Do they overwrite each other? I imagine this might be used if folks want to compare two versions, for example.

**Review by candiduslynx — APPROVED** [review]

**Review by hermanschaaf — APPROVED** [review]
Works like a charm!

File: cli/cmd/tables.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+	source "github.com/cloudquery/plugin-sdk/clients/source/v1"
+	"github.com/cloudquery/plugin-sdk/specs"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+)
+
+const (
+	tablesShort  = "Generate documentation for all supported tables of source plugins specified in the spec(s)"
+	ablesExample = `# Generate documentation for all supported tables of source plugins specified in the spec(s)
```
Comment (disq) [inline cli/cmd/tables.go:16]:
```suggestion
	tablesExample = `# Generate documentation for all supported tables of source plugins specified in the spec(s) 
```

File: cli/cmd/tables.go [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+cloudquery tables ./directory --output-dir ./docs
+`
+)
+
+func NewCmdTables() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "tables [files or directories]",
+		Short:   tablesShort,
+		Long:    tablesShort,
+		Example: ablesExample,
```
Comment (disq) [inline cli/cmd/tables.go:30]:
```suggestion
		Example: tablesExample,
```

**Pull request merged**